### PR TITLE
Docs cleanup: fix remaining line breaks, fix a plethora of typos, add several helpful links

### PR DESCRIPTION
**Pull request by cvan**
r? @ngokevin @jcarpenter

### CODE REVIEW

File: src/docs/components/cursor.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ parent_section: components
 order: 2
 ---
 
-The cursor component defines interaction with the scene through clicking and gazing, by using a raycaster to determine which object has been clicked. When the mouse is clicked, the closest visible entity intersecting the cursor will have a DOM click event triggered.
+The `cursor` component defines interaction with the scene through clicking and gazing, by using a [raycaster](https://en.wikipedia.org/wiki/Ray_casting) to determine which object has been clicked. When the mouse is clicked, the closest visible entity intersecting the cursor will have a DOM `click` event triggered.
```
Comment (ngokevin) [inline src/docs/components/cursor.md]: alternatively, link to raycaster component, and link to wiki from raycaster component

File: src/docs/components/geometry.md [hunk 2]
```diff
@@ -125,9 +112,7 @@ specified on the material component.
 
 ### Ring
 
-The ring geometry defines a flat ring, like a CD. Note that because it is
-flat, only a single side of the ring will be rendered if "side: double" is
-not specified on the material component.
+The ring geometry defines a flat ring, like a [CD](https://en.wikipedia.org/wiki/Compact_disc). Note that because it is flat, only a single side of the ring will be rendered if `side: double` is not specified on the `material` component.
```
Comment (ngokevin) [inline src/docs/components/geometry.md]: feel old moment when we have to explain a CD

Comment (cvan) [inline src/docs/components/geometry.md]: lol :cd:

File: src/docs/components/scale.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Note that the scaling factors cannot be `0` or else the matrix transformations i
 
 Scaling factors can be negative, which results in a reflection.
 
-A notable use for this is for sky spheres. Sky spheres contain the entire scene and have a texture mapped on the interior surface. To accomplish this, we can reflect, or invert, the sphere in the Z-direction.
+A notable use for this is for sky spheres. Sky spheres contain the entire scene and have a texture mapped on the interior surface. To accomplish this, we can reflect, or invert, the sphere in the Z direction.
```
Comment (ngokevin) [inline src/docs/components/scale.md]: https://en.wiktionary.org/wiki/z-direction

Comment (cvan) [inline src/docs/components/scale.md]: yeah, I saw that in Google but that was the only result

File: src/docs/components/cursor.md (hunk 1 above)
Comment (cvan) [inline src/docs/components/cursor.md]: I thought about it, but `cursor` doesn't actually use the `raycaster` component. but changed anyway.

File: src/docs/components/scale.md (hunk 3 above)
Comment (cvan) [inline src/docs/components/scale.md]: also: https://github.com/aframevr/aframe-site-dev/pull/93/files#diff-21851ce54cca91b44f36964cfbf94db8R23

**Comment by cvan** [comment]
rr? @ngokevin - updated, thanks

File: src/docs/components/cursor.md (hunk 1 above)
Comment (ngokevin) [inline src/docs/components/cursor.md]: The cursor component depends on the ray caster component

Comment (cvan) [inline src/docs/components/cursor.md]: ah, actually uses `dependencies` - gotcha. I thought the raycaster code was just shared

**Pull request merged**